<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterView</class>
 <widget class="QWidget" name="MasterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MasterView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#stackedWidget {
    background-image: url(:/image/R21.png); /* 背景图片路径 */
    background-repeat: no-repeat; /* 不重复图片 */
    background-position: center; /* 图片居中 */
}
QLineEdit {
    background: transparent; /* 设置背景透明 */
    border: 2px solid #e0e0e0; /* 边框颜色 */
    border-radius: 5px; /* 边框圆角 */
    padding: 5px;
    color: #34495e; /* 输入文字颜色 */
    font: 12px &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid  #a0a0a0; /* 聚焦时的边框颜色 */
}

QPushButton {
    background-color: #e0e0e0; /* 浅灰色背景 */
    color: #ffffff; /* 白色文字 */
    padding: 10px 20px;
    border: none; /* 去掉默认边框 */
    border-radius: 5px;
}
QLabel {
color:white;
    font: bold 20px &quot;Arial&quot;;
}

/* 按钮悬停时的颜色 */
QPushButton:hover {
    background-color: #a0a0a0; /* 深灰色背景 */
    color: #ffffff; /* 保持白色文字 */
}

/* 按钮按下时的颜色 */
QPushButton:pressed {
    background-color: #7f7f7f; /* 中灰色背景 */
    color: #ffffff; /* 保持白色文字 */
}
QTableView { 
      color: white;               
    background-color: #2e2e2e; 
       gridline-color: #555; 
    } 
    QHeaderView::section { 
       background-color: #3e3e3e; 
  color: white; 
    } 
   QTableView::item:selected { 
    background-color: #6a6a6a; 
  color: white;
    }
 QTableView::item { 
    background-color: black; 
  color: white;
    }

QWidget { font-family: 'Arial'; font-size: 18px; font-weight: bold; color: black; }
#MasterView{background-color:#d1d1d1}



</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btBack">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="lab3.qrc">
         <normaloff>:/image/return (1).png</normaloff>:/image/return (1).png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btLogout">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="lab3.qrc">
         <normaloff>:/image/logout.png</normaloff>:/image/logout.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lab3.qrc"/>
 </resources>
 <connections/>
</ui>
